flake.nix

```diff
@@ -77,6 +77,11 @@
       flake = false;
     };
 
+    blockfrost-platform-service = {
+      url = "github:blockfrost/blockfrost-platform/pull/273/head";
```

Comment:
Now that 273 is merged, would you prefer to bump this to `github:blockfrost/blockfrost-platform` and update the lock?

Reply:
Fixed in bd48d2c9147d093086774277061da0e158c544b0.